ppl/lake/schema.go

```diff
@@ -196,6 +196,11 @@ func CreateOrOpenLakeWithContext(ctx context.Context, rpath string, co *CreateOp
 	mdPath := root.AppendPath(metadataFilename)
 	ok, err := iosrc.Exists(ctx, mdPath)
 	if err != nil {
+		// The error encountered here has been an S3 permission failure,
+		// and it is fatal, so a panic might be better.
+		// A log message would be good in any case. Providing access to
+		// a logger will require modifying mulitple layers of APIs, so we
+		// should discuss the best solution first. -Mark
```

Comment:
@alfred-landrum created https://github.com/brimsec/zq/issues/2204 for this. If additional work is necessary, let's create another issue instead of adding notes like this.
```suggestion
		// The error encountered here has been an S3 permission failure,
		// and it is fatal, so a panic might be better.
		// A log message would be good in any case. Providing access to
		// a logger will require modifying mulitple layers of APIs, so we
		// should discuss the best solution first. -Mark
```

Reply:
Agreed, or at least this should go in another pr.

Reply:
This code location is significant. It will be source a failures in the future. The issue 2204 is generic and does not refer to this specific code.

Reply:
I will create an issue that refers to this location and remove the comment.